monai/inferers/inferer.py

```diff
@@ -100,34 +100,29 @@ class PatchInferer(Inferer):
         output_keys: if the network output is a dictionary, this defines the keys of
             the output dictionary to be used for merging.
             Defaults to None, where all the keys are used.
+        match_spatial_shape: whether to crop the output to match the input shape. Defaults to True.
         merger_kwargs: arguments to be passed to `merger_cls` for instantiation.
-            `output_shape` is calculated automatically based on the input shape and
+            `merging_shape` is calculated automatically based on the input shape and
             the output patch shape unless it is passed here.
     """
 
     def __init__(
         self,
-        splitter: Splitter | Callable | None = None,
+        splitter: Splitter | None = None,
         merger_cls: type[Merger] | str = AvgMerger,
         batch_size: int = 1,
         preprocessing: Callable | None = None,
         postprocessing: Callable | None = None,
         output_keys: Sequence | None = None,
+        match_spatial_shape: bool = True,
         **merger_kwargs: Any,
     ) -> None:
         Inferer.__init__(self)
-
         # splitter
         if splitter is not None and not isinstance(splitter, Splitter):
```

Comment:
```suggestion
        if not isinstance(splitter, (Splitter, type(None))):
```
and remove the enclosed `if not isinstance(splitter, Splitter):` as redundant.